src/util/to-conventional-changelog-format.ts

```diff
@@ -183,9 +189,60 @@ export default function toConventionalChangelogFormat(
     );
     // TODO(@bcoe): how should references like "Refs: v8:8940" work.
     if (hasRefSepartor && reference.issue.match(NUMBER_REGEX)) {
-      cc.references.push(reference);
+      headerCommit.references.push(reference);
     }
   });
 
-  return cc;
+  /*
+   * Split footers that resemble commits into additional commits, e.g.,
+   * chore: multiple commits
+   * chore(recaptchaenterprise): migrate recaptchaenterprise to the Java microgenerator
+   *   Committer: @miraleung
+   *   PiperOrigin-RevId: 345559154
+   * ...
+   */
+  visitWithAncestors(
+    ast,
+    ['type'],
+    (node: parser.Type, ancestors: parser.Node[]) => {
+      let parent = ancestors.pop();
+      if (!parent) {
+        return;
+      }
+      if (parent.type === 'token') {
+        parent = ancestors.pop();
+        let footerText = '';
+        visit(
+          parent,
+          ['type', 'scope', 'breaking-change', 'separator', 'text', 'newline'],
+          (node: FooterNodes) => {
+            switch (node.type) {
+              case 'scope':
+                footerText += `(${node.value})`;
```

Comment:
This is necessary because we don't currently include `(` and `)` in the parser, I'd like to add a `parenthesis` node, which would simplify logic.